drivers/bridge/setup_ipv6.go

```diff
@@ -133,17 +132,12 @@ func setupNDPProxying(config *networkConfiguration, i *bridgeInterface) error {
 			logrus.Warnf("Unable to enable NDP default proxying: %v", err)
 		}
 	}
-
-	// Get current NDP all proxying setup
-	ndpProxyDataAll, err := ioutil.ReadFile(ndpProxyConfAll)
```

Comment:
I see in most of bridge code we first check the flag value, and we set it to 1 only if needed. 
I know we would do only one write if we set it regardless, but I think it is better to be consistent and check first.